experimental/python/databricks/bundles/build.py

```diff
@@ -0,0 +1,479 @@
+        raise ValueError("Missing required argument --output")
+
+    if not parsed.diagnostics:
+        raise ValueError("Missing required argument --diagnostics")
+
+    return _Args(
+        phase=parsed.phase,
+        input=parsed.input,
+        output=parsed.output,
+        diagnostics=parsed.diagnostics,
+        locations=parsed.locations,
+        unknown_args=unknown_args,
+    )
+
+
+def _load_resources_from_input(input: dict) -> tuple[Resources, Diagnostics]:
+    resources = Resources()
+    diagnostics = Diagnostics()
+
+    input_resources = input.get("resources", {})
+    input_jobs = input_resources.get("jobs", {})
+
+    for resource_name, job_dict in input_jobs.items():
+        try:
+            job = Job.from_dict(job_dict)
+
+            resources.add_job(resource_name, job)
+        except Exception as exc:
+            diagnostics = diagnostics.extend(
+                Diagnostics.from_exception(
+                    exc=exc,
+                    summary="Error while loading job",
+                    path=("resources", "jobs", resource_name),
+                )
+            )
+
+    return resources, diagnostics
+
+
+def _apply_mutators(
+    bundle: Bundle,
+    resources: Resources,
+    mutator_functions: list[ResourceMutator],
+) -> tuple[Resources, Diagnostics]:
+    for resource_name, job in resources.jobs.items():
+        for mutator in mutator_functions:
+            if mutator.resource_type != Job:
+                continue
+
+            location = Location.from_callable(mutator.function)
+
+            try:
+                if _get_num_args(mutator.function) == 1:
+                    new_job = mutator.function(job)
+                else:
+                    # defensive copy so that one function doesn't affect another
+                    new_job = mutator.function(deepcopy(bundle), job)
+
+                # mutating job in-place works, but we can't tell when it happens,
+                # so we only update location if new instance is returned
+
+                if new_job is not job:
+                    if location:
+                        resources.add_location(
+                            ("resources", "jobs", resource_name), location
+                        )
+                    resources.jobs[resource_name] = new_job
+            except Exception as exc:
+                mutator_name = mutator.function.__name__
+
+                return resources, Diagnostics.from_exception(
+                    exc=exc,
+                    summary=f"Failed to apply '{mutator_name}' mutator",
+                    location=location,
+                    path=("resources", "jobs", resource_name),
+                )
+
+    return resources, Diagnostics()
+
+
+def python_mutator(
+    args: _Args,
+) -> tuple[dict, dict[tuple[str, ...], Location], Diagnostics]:
+    input = json.load(open(args.input))
+    experimental = input.get("experimental", {})
+
+    if experimental.get("pydabs", {}) != {}:
```

Comment:
This can be removed.

Reply:
It will help with migration if the wrong package is used. What if we remove it together with property support?

Reply:
Sounds good. This will also simplify the Go code.